### DISCUSSION TITLE
how to reset global cache in kong ?

### DISCUSSION THREAD

**Discussion by shilaidun** [Kong/kong#12319] [Help] [answered]
I wrote a custom plugin to use the local cache. When I received a request that needed to update the local cache, Kong only updated the cache of the current worker process and did not apply it to other worker processes.

**Comment by Water-Melon** [comment]
I think [`kong.cache`](https://docs.konghq.com/gateway/latest/plugin-development/pdk/#kongcache) or [`ngx.shared`](https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#ngxshareddict) can meet your needs.

**Comment by ms2008** [comment, marked as answer]
You can purge Kong's cache this way: `curl -X DELETE http://localhost:8001/cache`. Also `kong.cache.invalidate_local()` only invalidates the current worker, you may need `kong.cache.invalidate()` if you want to invalidate all workers.
